tests/integration/targets/inventory/playbooks/sysparm_limit.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Test sysparm_limit option
+  hosts: localhost
+  gather_facts: false
+
+  tasks:
+    - block:
```

Comment:
Please use `sysparm_limit` in your test. This is the functionality we want to test.